### PR TITLE
Disable Flower by default from docker-compose

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragement file, named `{pr_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: docs/apache-airflow/executor/kubernetes.rst [hunk 1]
```diff
@@ -214,7 +214,9 @@ To get task logs out of the workers, you can:
 Comparison with CeleryExecutor
 ------------------------------
 
-In contrast to CeleryExecutor, KubernetesExecutor does not require additional components such as Redis and Flower, but does require access to Kubernetes cluster.
+In contrast to CeleryExecutor, KubernetesExecutor does not require additional components such as Redis, but does require access to Kubernetes cluster.
+
+Also monitoring the Pods can be done with the built-in Kubernetes monitoring. For Celery you might want to use tools like Flower to monitor running Celery workers.
```
Comment (jedcunningham) [inline docs/apache-airflow/executor/kubernetes.rst]:
```suggestion
Also monitoring the Pods can be done with the built-in Kubernetes monitoring.
```

Probably better to add that to the celery page instead? Seems out of place here to me. Maybe even bring this into the paragraph above?

Comment (potiuk) [reply docs/apache-airflow/executor/kubernetes.rst]: Crossed my mind too :)

Comment (potiuk) [reply docs/apache-airflow/executor/kubernetes.rst]: The less we mention Flower, the better. Removed it from here simply.

**Review by jedcunningham — APPROVED** [review]

**Review by kaxil — APPROVED** [review]

**Comment by potiuk** [comment]
Cherry-picked to 2.3.1

**Pull request merged**